Formula/mdxmini.rb

```diff
@@ -1,6 +1,6 @@
 class Mdxmini < Formula
   desc "Plays music in X68000 MDX chiptune format"
-  homepage "http://clogging.web.fc2.com/psp/"
+  homepage "https://clogging.web.fc2.com/psp/"
```

Comment:
It's valid, but it's a redirect to HTTP indeed. We used to consider these cases valid candidates for an URL upgrade. Probably something to be fixed on the webserver though, similarly to mixed-content issues.